iterator_practice.cpp

```diff
@@ -1,16 +1,13 @@
 #include <iostream>
-#include <iterator> // for iterator
 #include <vector> // for vectors 
 
-using namespace std;
-
-int main()
```

Comment:
I made the style changes we usually use: 2 spaces to indent, blank line at the end of the file, opening braves on the same line, etc.

It's not worth sweating this yourself. Instead, if you add a copy of .clang_format from the top of the spectre repo to your repo, you can just run `clang-format -I file.cpp` to enforce our formatting rules